packages/patternfly-4/react-core/src/components/LoginPage/LoginMainHeader.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Title } from '../Title';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly-next/components/Login/login.css';
+
+const propTypes = {
+  /** Content rendered inside the Login Main Header */
+  children: PropTypes.node,
+  /** Additional classes added to the Login Main Header */
+  className: PropTypes.string,
+  /** Dropdown component for the Login Main Header */
+  dropdown: PropTypes.node,
+  /** Title for the Login Main Header */
+  title: PropTypes.string,
+  /** Subtitle that contains the Text, URL, and URL Text for the Login Main Header */
+  subtitle: PropTypes.node
+};
+
+const defaultProps = {
+  children: null,
+  className: '',
+  dropdown: null,
+  title: '',
+  subtitle: ''
+};
+
+const LoginMainHeader = ({ children, className, dropdown, title, subtitle, ...props }) => (
+  <header className={css(styles.loginMainHeader, className)} {...props}>
+    <Title size="3xl">{title}</Title>
```

Comment:
For optional elements, wonder if we need to surround with conditional, in case it takes up space in the DOM if unused, e.g. `{title && <Title...`